subprojects/docs/src/docs/release/notes.md

```diff
@@ -7,6 +7,7 @@ We would like to thank the following community members for their contributions t
 Include only their name, impactful features should be called out separately below.
  [Some person](https://github.com/some-person)
 -->
+[Shou Kakerigawa(翳河翔)](https://github.com/anatawa12)
```

Comment:
Can you add as anatawa12? Shou Kakerigawa is also a handle name but not for programming.

Reply:
Of course!  I'll merge that change for you.